deadcode/editor.js

```diff
@@ -2558,14 +2558,14 @@ const loadHash = async hash => {
     loadedObject = null;
   }
 
-  const u = `${storageHost}/ipfs/${hash}/.metaversefile`;
+  const u = `${storageHost}/ipfs/${hash}/.totum`;
   loadedObject = await world.addObject(u);
   loadedObject.name = loadedObject.contentId.match(/([^\/]*)$/)[1];
 };
 const mintNft = async () => {
   const hash = await uploadHash();
   console.log('upload nft', hash);
-  window.location.href = `https://webaverse.com/mint?hash=${hash}&ext=metaversefile`;
+  window.location.href = `https://webaverse.com/mint?hash=${hash}&ext=totum`;
```

Comment:
I don't think this part is right. We should not be renaming the actual file IMO. This is literally the file type, which can stay as `metaversefile`.

Reply:
got it ok, no problem